src/app/admin/Smaregi/Index.vue

```diff
@@ -31,68 +31,97 @@
+                >
+                  {{ threshold.name }}
+                </option>
+              </o-select>
+            </div>
+
+            <div class="pl-4 pt-1 pb-4">
+              在庫復活しきい値:
+              <o-select v-model="inStockData[k]">
+                <option
+                  v-for="threshold in inStockThresholds"
+                  :value="threshold.value"
+                  :key="threshold.value"
+                >
+                  {{ threshold.name }}
+                </option>
+              </o-select>
+            </div>
           </div>
           <div v-if="isDuplicateError">*お店の指定が重複しています</div>
-          <o-button @click="saveShops" :disabled="isDuplicateError"
-            >保存</o-button
-          >
+          <div class="mt-4">
+            <button @click="saveShops" :disabled="isDuplicateError">
+              <div
+                class="inline-flex h-12 items-center justify-center rounded-full bg-op-teal px-6 shadow"
+                style="min-width: 8rem"
+              >
+                <span class="text-base font-bold text-white">{{$t("editCommon.save")}}</span>
+              </div>
+            </button>
+          </div>
         </div>
 
         <div v-else>
-          <div v-for="(shop, k) in shopList" :key="k">
-            <div class="mt-2 text-base font-bold">
-              <router-link :to="`/admin/smaregi/store/${shop.storeId}`">
-                {{ shop.storeName }}
-              </router-link>
+          <div v-for="(shop, k) in shopList" :key="k"  class="mt-2 rounded-lg bg-black bg-opacity-5">
+            <div class="pl-4 pt-4">
+              <div class="text-base">
+                スマレジ登録店舗：
+                <router-link :to="`/admin/smaregi/store/${shop.storeId}`">
+                  <span class="font-bold">{{ shop.storeName }}</span>
+                </router-link>
+              </div>
+            </div>
+            <div class="pl-4">
+              連携店舗：{{ (restaurantObj[selectedRestaurant[k]] || {}).restaurantName}}
+            </div>
+            <div class="pl-4 pb-4">
+              在庫切れしきい値：
+              {{ showStockThreshold((outOfStockData || {})[k]) }} /
+              在庫復活しきい値：{{ showStockThreshold((inStockData || {})[k]) }}
             </div>
-            {{ (restaurantObj[selectedRestaurant[k]] || {}).restaurantName
-            }}<br />
-            在庫切れしきい値:
-            {{ showStockThreshold((outOfStockData || {})[k]) }} /
-            在庫復活しきい値: {{ showStockThreshold((inStockData || {})[k]) }}
           </div>
           <div class="mt-4">
-            <o-button @click="isEdit = true">編集</o-button>
+            <button @click="isEdit = true">
+              <div
+                class="inline-flex h-12 items-center justify-center rounded-full bg-op-teal px-6 shadow"
+                style="min-width: 8rem"
+              >
+                <span class="text-base font-bold text-white">編集</span>
+              </div>
+            </button>
```

Comment:
```suggestion
                style="min-width: 8rem"
```

上記と同様に `min-w-32` を使うのが良いと思います